Doc/library/sys.rst

```diff
@@ -1353,7 +1353,7 @@ always available.
       This function has been added on a provisional basis (see :pep:`411`
       for details.)  Use it only for debugging purposes.
 
-   .. deprecated:: 3.7
+   .. deprecated-removed:: 3.7 3.8
```

Comment:
Side question:
Do we really want to remove it in 3.8?
If yes -- the time window is very close.
@1st1 we need your judgment for it.

Reply:
I think we should drop it. asyncio no longer needs it, AFAIK Trio considered using it but decided it's too slow.

Reply:
Nice!
Thank you very much!

Reply:
@1st1  I've opened https://bugs.python.org/issue36933 yesterday to track the removal.